### DISCUSSION TITLE
Game Vision Discussion

### DISCUSSION THREAD

**Discussion by doubledh** [Imaginary-Horizons-Productions/prophets-of-the-labyrinth-Archive#416] [Ideas]
I propose throwing out a bunch of "W" questions (who/what/when/where/why) to see if we can carve out what the lore/background/intention of the game. We can try a choose-your-own/ad-lib adventure kind of approach to building the game vision, where we riff on each other's ideas or vet them (hopefully without griefing others' works often). Many questions will also feature an optional "Why"

**Comment by doubledh** [comment]
Who are the protags? (The Prophets?)
Why are they Prophets? Are they actually part of a group?
Why are they the good guys?

**Reply by ntseng** [reply]
~~I sorta imagined the prophets being summoned to the aid of someone exploring/stopping badness from happening in the nearby labyrinth. Thus, the particular individuals are summoned were based on the selection of the summoners.~~

~~I didn't really have much story in mind for each of the archetypes, and mostly wanted them to express strong gameplay themes.~~

**Reply by doubledh** [reply]
Prophets by definition speak about the will of a deity... I'm actually surprised that prophecies themselves are not always about the future, per se, but are just written/spoken will on a deity's behalf.

That said, we could suppose there be some deity that the prophets fight for.

**Reply by ntseng** [reply]
I actually like that better, divine quest being the motivation and divine blessing being the explanation of power.

**Reply by doubledh** [reply]
Prophets have their Predicts given to them as blessings from their diety(/ies)?

**Comment by doubledh** [comment]
Who are the Antags? (The end-bosses?)
Why are they the bad guys?

**Reply by ntseng** [reply]
~~Not really sure on this one yet. Most of the final bosses stemmed from gameplay ideas. Settings wise, the clones are magical trap, and the Elkemist just wants to be left alone.~~

~~Who set up these traps and why are they bad? Not sure.~~

**Reply by ntseng** [reply]
If divine quest is the motivation, that would mean the final bosses are abhorrent to the quest giving deities?

**Comment by doubledh** [comment]
Who are the supporting cast? (the npc's?)
Why do they support the Protags?

**Reply by ntseng** [reply]
~~This would likely be the summoners and their cohort.~~

This would likely be whatever cohort accompanies on the divine quest.

**Reply by ntseng** [reply]
I'm pretty on the fence on if we want to do allied AI. If we do, it'd be really cool if the allies also made decisions restricted by the predict information available in the party.

On the other hand though, I can't think of many cases where people really love working with the allied AI. So we might not get a lot of bang for our buck on that work.

**Reply by bobaguardian** [reply]
I really like the divine quest theme!  Just throwing stuff out there lore-wise, but maybe the NPCS (merchants/forge areas/rest sites/etc) could be dwellers in the specific labyrinth that need the powers of the prophets and deities to "purify" or rid their labyrinth of these monsters and bosses (and each boss can have their own motive/description) -- I know the labyrinth names are chosen randomly but each can have a sort of theme.

**Reply by ntseng** [reply]
The random picking of labyrinth names were more me checking that different seeds were being generated than anything I was really attached to. Definitely down to have them be more curated/informative of their contents!

**Comment by doubledh** [comment]
Who are the obstructing cast? (the mobs?/ bad event npc's?)
Why do they hinder the Protags?

**Reply by ntseng** [reply]
Most of the normal enemies seem to be wildlife so far. Maybe we should think about an opposing faction or something?

**Comment by doubledh** [comment]
When does gameplay take place, when a group of prophets attempts to delve? (present)
What came before that? (historical lore)
Would there be a canonical ending or lack-of-an-ending, after each or the canonical-final delve? (future end-endgame state)

**Reply by ntseng** [reply]
The main timing logic is players collecting artifacts. If the player brings an artifact, that delve probably happened after the one they collected the artifact on (at least in the player's time frame).

We could also have some of the more difficult labyrinths canonically be explored after the easier ones.

**Comment by doubledh** [comment]
Where does the gameplay take place? (the labyrinth?)
Is the labyrinth the only place of interest, and does it have anything around it (that could reasonably be referenced during gameplay)?
What is this labyrinth like, as an experience? (sensory elaboration)

**Reply by ntseng** [reply]
One feature I've been thinking about is having different labyrinths by making different sets of rooms. This would allow different themes and difficulty selection.

**Reply by doubledh** [reply]
I subscribe to the idea of multiple labyrinths

**Comment by doubledh** [comment]
Who is the target audience of the game?  (Conversely, how accessible/inclusive need our game be?)

**Reply by ntseng** [reply]
I think the main genre is definitely roguelite. Players who like exploring and adapting to different situations should like this game.

One big distinction is that most roguelites are single-player. I'd really like to support "adding to the group power, rather than just my own" as part of the multiplayer aspect of this game.

I have no idea how large of an audience demographic this intersection represents, and the untested demographic is one of my reservations for pushing the project for profitability.

**Reply by ntseng** [reply]
I think the divine quest theme gives us an interesting opportunity to not be as gritty and morally ambiguous as most roguelites tend to theme themselves.

I'm not sure I can write a game that takes itself completely seriously though. Here's an expert from a previously removed room:
`
You stumble upon what appears to be a brute convention. The keynote brute interrupts their presentation on \"How to be Faceless\" to point out that a practical opportunity to work on mugging skills has just entered the room.
`

**Comment by doubledh** [comment]
How many ways (designed) strategies should there be, to playing this game?

**Reply by doubledh** [reply]
I appreciated how in Slay the Spire, there were several characters and several playstyles WITHIN each character, to build for. I'm a fan of designing at least 2 axes... of maybe  2-4 correlated build choices (somewhere around maybe 8-16 designed playstyles)

**Reply by ntseng** [reply]
Branching upgrades is supposed to get at this, but it'll probably take a few balance passes/reorganizations before those really come together.

**Reply by ntseng** [reply]
We should also have some stuff that's aimed at being more emergent. If builds feel too designed, it cuts out a certain fun of "breaking the game".

**Comment by doubledh** [comment]
How difficult should the game be?

**Reply by doubledh** [reply]
As a player, I personally like the idea of being able to relatively set the difficulty. I suppose there can be a problem where difficulty is set to infantile... but even from seeing super adventure box, infantile mode wasn't exactly a loss

**Reply by ntseng** [reply]
I'd like a beginning player taking no difficulty options to complete most seeds. I'd like an experienced player taking all the difficulty options to complete very few seeds.

**Comment by doubledh** [comment]
What are the rewards of playing the game? What are the rewards of progressing in the game?  What are the rewards of "beating" the game?

**Reply by ntseng** [reply]
So far I can think of 3 (planned):
- collecting artifacts
- rolling for more archetypes
- pushing one's high scores

**Comment by doubledh** [comment]
How serious/sensible should this game be? Should everything follow some logical canon, or can stuff be wacky/cartoony/fourth-wall-breaky?

**Reply by doubledh** [reply]
I was under the impression that the game was pretty free to be able to break as many rules as it wants to if it wants to be zany

**Reply by ntseng** [reply]
I'm all for not taking ourselves too seriously. My main concern is that the jokes be ignorable on subsequent plays as to not get annoying.
